### ISSUE TITLE
Use attribute "aria-label" in popup panel to be assistive tech-friendly

### ISSUE THREAD

**Issue by gorhill** [gorhill/uBlock#2889]
About `aria-label`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

> The aria-label attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen.

From uBO's point of view, whether the attribute is named `data-tip` or `aria-label` makes no difference, whereas the latter can be used by screen readers.

**Comment by gorhill** [comment]
Turns  out there was already issue #2072 out there, I had forgotten about it.
